packages/rocketchat-ui-flextab/client/tabs/membersList.js

```diff
@@ -76,7 +81,7 @@ Template.membersList.helpers({
 		}
 		// show online users first.
 		// sortBy is stable, so we can do this
-		users = _.sortBy(users, u => u.status == null);
+		users = _.sortBy(users, u => u.status ===	 'offline');
```

Comment:
Can you fix this spacing?

As a note, why exactly is this in this pull request?

Reply:
because I was working on member list and I saw that problem... maybe I should made an special  commit for that